Try out PhantomJS 2.5 beta

See https://github.com/ariya/phantomjs/issues/14458 / https://github.com/Medium/phantomjs/issues/668 / https://github.com/macbre/phantomjs2-npm/issues/3

> Please note that PhantomjJS 2.5 may **[never reach the stable state](https://news.ycombinator.com/item?id=13350728)**

* [x] make https://github.com/macbre/phantomjs-static-2.5/releases/tag/v2.5-beta use static binary
* [x] use forked `phantomjs-static-2.5` npm module

## New features

* added support for WebP - https://gist.github.com/macbre/03efa4479ab80dca4437#file-results-md

```
echo "Recompiling OpenSSL for ${OPENSSL_TARGET}..." && sleep 1
....

installing man1/cms.1ssl
cms.pod around line 457: Expected text after =item, not a number
cms.pod around line 461: Expected text after =item, not a number
cms.pod around line 465: Expected text after =item, not a number
cms.pod around line 470: Expected text after =item, not a number
cms.pod around line 474: Expected text after =item, not a number
POD document had syntax errors at /usr/bin/pod2man line 71.
make: *** [install_docs] Error 255
Makefile:640: recipe for target 'install_docs' failed
```

--- a/README.md
+++ b/README.md
@@ -11,12 +11,12 @@ PhantomJS-based modular web performance metrics collector. And why phantomas? We
 ## Installation
 
 ```
-npm install --global --no-optional phantomas phantomjs-prebuilt@^2.1.5
+npm install --global --no-optional phantomas phantomjs-prebuilt@macbre/phantomjs2-npm#v2.5.0-dev
 ```
 
-> This will install the latest version of PhantomJS and add a symlink called ``phantomas`` (pointing to ``./bin/phantomas.js``) to your system's ``PATH``
+> This will install [the latest version of PhantomJS](https://github.com/macbre/phantomjs-static-2.5/releases/tag/v2.5-beta) and add a symlink called ``phantomas`` (pointing to ``./bin/phantomas.js``) to your system's ``PATH``
 
-You may need to install libfontconfig by running ``sudo apt-get install libfontconfig1``.
+You may need to install libfontconfig and libjpeg8 by running ``sudo apt-get install libfontconfig1 libjpeg8``.
 
 ### Development version
 

--- a/core/phantomas.js
+++ b/core/phantomas.js
@@ -438,6 +438,11 @@ phantomas.prototype = {
 		this.log('Using %s as user agent', this.page.settings.userAgent);
 		this.log('Viewport set to %d x %d', this.page.viewportSize.width, this.page.viewportSize.height);
 
+		if (typeof phantom !== 'undefined') {
+			this.log('phantom.version: %j (%s)', phantom.version);
+			this.log('phantom user agent: %s', phantom.defaultPageSettings.userAgent);
+		}
+
 		// bind basic events
 		this.page.onInitialized = this.proxy(this.onInitialized);
 		this.page.onLoadStarted = this.proxy(this.onLoadStarted);

--- a/core/phantomas.js
+++ b/core/phantomas.js
@@ -439,7 +439,7 @@ phantomas.prototype = {
 		this.log('Viewport set to %d x %d', this.page.viewportSize.width, this.page.viewportSize.height);
 
 		if (typeof phantom !== 'undefined') {
-			this.log('phantom.version: %j (%s)', phantom.version);
+			this.log('phantom.version: %j', phantom.version);
 			this.log('phantom user agent: %s', phantom.defaultPageSettings.userAgent);
 		}
 

--- a/package.json
+++ b/package.json
@@ -48,7 +48,7 @@
     "vows": "^0.7.0"
   },
   "optionalDependencies": {
-    "phantomjs-prebuilt": "^2.1.14",
+    "phantomjs-prebuilt": "macbre/phantomjs2-npm#v2.5.0-dev ",
     "slimerjs": "^0.10.3"
   },
   "bin": "./bin/phantomas.js",

--- a/package.json
+++ b/package.json
@@ -48,7 +48,7 @@
     "vows": "^0.7.0"
   },
   "optionalDependencies": {
-    "phantomjs-prebuilt": "macbre/phantomjs2-npm#v2.5.0-dev ",
+    "phantomjs-prebuilt": "macbre/phantomjs2-npm#v2.5.0-dev",
     "slimerjs": "^0.10.3"
   },
   "bin": "./bin/phantomas.js",

